.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
I suggest caching only the ``/files`` subfolder. See https://github.com/puli/manager/pull/9#discussion_r40499480 for the explanation